spec/spec.md

```diff
@@ -0,0 +1,180 @@
+individual file in a project.
+
+## Definitions
+
+These are the definitions for some of the terms used in this specification:
+
+- Project --- ...
+
+- License file --- ...
+
+<!-- Also put a link to the SPDX Specification here somewhere -->
+
+<!-- Also DEP5 -->
+
+## License files
+
+A project must include a license file for every license under which files in the
+project are licensed.
+
+Each license file should be placed in the `LICENSES/` directory in the root of
+the project. The name of the license file should be the SPDX identifier of the
+license followed by a file extension (example: `GPL-3.0-or-later.txt`).
+
+<!-- FIXME: Rephrase "above recommendation". Maybe use paragraph numbers? -->
+
+A license file whose name does not conform to the above recommendation must
+contain, at the start of the file, one or more `Valid-License-Identifier` tags,
+followed by a `License-Text` tag, followed by the license text. Example:
+
+```
+Valid-License-Identifier: GPL-3.0-only
+Valid-License-Identifier: GPL-3.0-or-later
+License-Text:
+
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+[...]
+```
+
+A project may contain a single license file outside of the `LICENSES/` directory
+in the file `COPYING`, `COPYRIGHT`, `LICENSE` or `LICENCE` in the root of the
+project.
+
+If a license does not exist in the SPDX License List, its SPDX identifier is
+`LicenseRef-[idstring]` as defined by the SPDX Specification.
+
+A project must not include license files for licenses under which none of the
+files in the project are licensed.
+
+Everything that applies to licenses in this section also applies to license
+exceptions, with the following exceptions:
+
+- Exception files must be placed in the `LICENSES/` directory in the root of the
+  project.
+
+- The tags `Valid-Exception-Identifier` and `Exception-Text` must be used
+  instead of `Valid-License-Identifier` and `License-Text`.
+
+<!-- Can we use LicenseRef for exceptions? -->
+
+<!-- TODO: Change "copyright and licensing information" to something shorter?
+Shorten it to CALI? -->
+
+## Copyright and licensing information
+
+Each file in the project must have copyright and licensing information
+associated with it, except the following files:
+
+- The license files.
+
+- The files belonging to the project's version control system (example:
+  `.git/`).
+
+- The files ignored by the version control system (example: `.gitignore`).
+
+- The files in the `.reuse/` directory in the root of the project.
+
+<!-- TODO: The files inside of a git submodule -->
+
+There are various ways to associate copyright and licensing information with a
+file.
+
+### Comment headers
+
+Each plain text file that can contain comments should contain comments at the
```

Comment:
```suggestion
To implement this method each plain text file that can contain comments MUST contain comments at the
```

Reply:
LGTM

Reply:
Fixed in c17e4d43b3a6b74ffa5de3004decae3348fba2c3